content/specs/versions4.md

```diff
@@ -70,14 +70,15 @@ weight: 2
 - **11.0.11** (default) JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home`
 - **1.8.0_292** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home`
 
-## Xcode 13.0 (13A5192j)
+## Xcode 13.0 (13A5201i)
 
 This is the Xcode version used by default when you select `13.0` or `edge` in build settings in the workflow 
 editor for Flutter apps or set Xcode version to `13.0` or `edge` in your codemagic.yaml file. 
+Other available versions are listed [here](#other-xcode-versions).

```

Comment:
@artemii-yanushevskyi I think this line is superfluous for Xcode 13.0 base image specs